snippets/graphql/get-metric-values-for-component/README.md

```diff
@@ -7,9 +7,9 @@ Replace `componentARI` below in the variables section with a valid [Compass comp
 ### Query
```

Comment:
On line 5 there is also a reference to `componentARI` that we may want to change

Reply:
Good catch! Thank you! Fixed it!